src/app/components/ComponentMap.tsx

```diff
@@ -367,7 +367,7 @@ export default function ComponentMap({
                               ? 'white'
                               : '#161521'
                         }
-                        z
+                        z={1}
```

Comment:
It's not clear to me why this attribute was originally there, or why it's needed.  So maybe a better fix is to remove it.